### PR TITLE
Lidarr support

### PR DESCRIPTION
**Pull request by badcrc**
Lidarr works like Sonarr and Radarr but for music.

https://lidarr.audio/

### CODE REVIEW

**Comment by tylergarcianet** [comment]
Nice to see this being included. I was able to use this to install Lidarr on my nas at home and then used your PR to model my after my PR to support Ombi.

File: group_vars/all.yml.dist [hunk 1]
```diff
@@ -43,6 +43,7 @@ bazarr_enabled: false
 # Music
 airsonic_enabled: false
 mymediaforalexa_enabled: false
+lidarr_enabled: false
```
Comment (tylergarcianet) [inline group_vars/all.yml.dist]: Might this fit better under `# Media Sourcing`?

File: tasks/lidarr.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+- name: Create lidarr Directories
+  file:
+    path: "{{ item }}"
+    state: directory
+  with_items:
+    - "{{ lidarr_data_directory }}"
```
Comment (tylergarcianet) [inline tasks/lidarr.yml]: You shouldn't need `with_items:` if we're only creating a single directory.

**Comment by davestephens** [comment]
Thanks for the contribution! (and your patience until I could find some time to merge this :+1:)

**Pull request merged**